src/mp2_grids.F

```diff
@@ -267,7 +279,30 @@ SUBROUTINE get_minimax_weights(para_env, unit_nr, homo, Eigenval, num_integ_poin
 
       CALL timestop(handle)
 
-   END SUBROUTINE get_minimax_weights
+   END SUBROUTINE get_minimax_grid
+
+! **************************************************************************************************
+!> \brief ...
+!> \param tj ...
+!> \param a_scaling ...
+! **************************************************************************************************
+   SUBROUTINE compute_tj_from_clenshaw(tj, a_scaling)
+      REAL(KIND=dp), ALLOCATABLE, DIMENSION(:), &
+         INTENT(INOUT)                                   :: tj
+      REAL(KIND=dp), INTENT(IN)                          :: a_scaling
+
+      CHARACTER(LEN=*), PARAMETER :: routineN = 'compute_tj_from_clenshaw', &
+         routineP = moduleN//':'//routineN
+
+      INTEGER                                            :: handle
+
+      CALL timeset(routineN, handle)
```

Comment:
Is it really necessary to time such a short function? What do you think of totally removing this helper function bc it is used nowhere else?

Reply:
Thanks for pointing this out, the function is removed.